docs/index.rst

```diff
@@ -478,6 +492,42 @@ introspection system (available in Pyramid 1.3+ only).
 
 .. image:: introspection.png
 
+.. _toolbar_settings:
+
+Toolbar Settings
+----------------
+
+The `Settings` "tab" allows for realtime customization of the toolbar and its
+panels.
+
+.. note::
+
+   This refers to the toolbar's `Settings` "tab" in the navigation bar, which is
+   distinct from the `Settings` "panel" (see :ref:`panel_settings`).
+
+.. image:: toolbar_settings.png
```

Comment:
Move this above the `seealso` revision.

Reply:
I'm not sure what this means. Do you mean just the image?

Reply:
Yes. It feels associated to the `seealso` because it is immediately after it, and it should be associated with the text above that.